generators/rabbitmq/files.mjs

```diff
@@ -10,39 +11,118 @@ import { moveToJavaPackageSrcDir } from '../server/support/index.mjs';
  */
 
 export const rabbitFiles = {
-    config: [
-      {
-        path: `${SERVER_MAIN_SRC_DIR}package/`,
-        renameTo: moveToJavaPackageSrcDir,
-        templates: ['config/RabbitMQConfig.java'],
-      },
-    ],
-    domain: [
-      {
-        path: `${SERVER_MAIN_SRC_DIR}package/`,
-        renameTo: moveToJavaPackageSrcDir,
-        templates: ['domain/RabbitMessageModel.java'],
-      },
-    ],
-    listener: [
-      {
-        path: `${SERVER_MAIN_SRC_DIR}package/`,
-        renameTo: moveToJavaPackageSrcDir,
-        templates: ['listener/RabbitMQConsumer.java'],
-      },
-    ],
-    publisher: [
+  config: [
+    {
+      path: `${SERVER_MAIN_SRC_DIR}package/`,
+      renameTo: moveToJavaPackageSrcDir,
+      templates: ['config/RabbitMQConfig.java'],
+    },
+  ],
+  domain: [
+    {
+      path: `${SERVER_MAIN_SRC_DIR}package/`,
+      renameTo: moveToJavaPackageSrcDir,
+      templates: ['domain/RabbitMessageModel.java'],
+    },
+  ],
+  listener: [
+    {
+      path: `${SERVER_MAIN_SRC_DIR}package/`,
+      renameTo: moveToJavaPackageSrcDir,
+      templates: ['listener/RabbitMQConsumer.java'],
+    },
+  ],
+  publisher: [
+    {
+      path: `${SERVER_MAIN_SRC_DIR}package/`,
+      renameTo: moveToJavaPackageSrcDir,
+      templates: ['publisher/RabbitMQPublisher.java'],
+    },
+  ],
+};
+
+export default async function writeRabbitMQFilesTask({ application }) {
+  if (communications.length === 0) {
+    application.queueName = application.baseName;
+    application.uniqueName = null;
```

Comment:
assigning null can cause some problem ryt?
why not initialise it with empty string 
application.uniqueName = "";